### DISCUSSION TITLE
SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by byronpolley** [system-ui/theme-ui#1280] [Q&A]
Hi all,

I have been having issues running a successful build on my app since upgrading to Next.js 10 and React 17.

I get this when running `yarn build`
```
import { jsx } from '@theme-ui/core'
^^^^^^
SyntaxError: Cannot use import statement outside a module
```

Any ideas how I can get successful builds again? I really don't want to migrate from theme-ui but I'm worried it won't be maintained. Again I know I don't get to say what happens with this repo but it's a consideration when adopting a package.

**Comment by lachlanjc** [comment]
Hey! Theme UI is very actively maintained. I haven’t gotten this issue & have a number of sites in production running Next.js 10, React 17, & Theme UI. Could you include a CodeSandbox/repl.it/etc with more context? Is this on a `pages` file you’re getting the error?

**Comment by byronpolley** [comment]
@lachlanjc Hi thanks for the info.

So I was getting this error whenever importing components: `Module '"../../node_modules/theme-ui/dist"' has no exported member 'Button'.ts(2305)`

So I added a `/src` to my input to correct it. Then it works but on build I get the above error. Will create a sandbox over the weekend if I don't come right.
